yt/yt/ytlib/api/native/client_authentication_impl.cpp

```diff
@@ -243,18 +261,29 @@ TListUserTokensResult TClient::DoListUserTokens(
     }
 
     std::vector<TString> userTokens;
+    std::map<TString, NYson::TYsonString> tokensMetadata;
```

Comment:
1) std::map -> THashMap
2) Not tokensMetadata, but tokenMetadata (by analogy with objectCount, not objectsCount).